collect_app/src/androidTest/java/org/odk/collect/android/TestSettingsProvider.kt

```diff
@@ -0,0 +1,21 @@
+package org.odk.collect.android
+
+import androidx.test.core.app.ApplicationProvider
+import org.odk.collect.android.application.Collect
+import org.odk.collect.android.injection.DaggerUtils
+import org.odk.collect.settings.SettingsProvider
+import org.odk.collect.shared.settings.Settings
+
+// Use just for testing
+object TestSettingsProvider {
```

Comment:
We have a separate class for android and unit tests. Does that mean it can't be shared?

Reply:
The old way of sharing (using a shared source set) isn't supported anymore. The `androidTest` one is actually only used in 3 places, so I can just replace those calls so we don't have the duplicated object.